tied-buffer/sycl-2.2/buffer-tags.md

```diff
@@ -74,41 +84,222 @@ class buffer {
 };
 ```
 
-However functions such as `get_context()` will only be valid if the `buffer` was constructed with the `tag::context_bound` tag so there must be a way for the user to query whether a `buffer` has a particular tag.
+However functions such as `get_context()` will only be valid if the `buffer` was constructed with the `property::context_bound` tag.
+The user can query for a specific property on any buffer by using the
+*has\_property* method.
 
 ```cpp
 template <typename dataT, int kElems, typename allocatorT = buffer_allocator<dataT>>
 class buffer {
   ...
 
-  bool has_tag(tag t)
+  bool has_property(tag t)
 
   ...
 };
 ```
 
+## Defined properties
+
+### Extension to SYCL 1.2
+
+* *map_ptr*: Buffer will use the given pointer exclusively for host access.
+
+```cpp
+struct map_ptr {
+
+  map_ptr() = default;
+
+  // Implementation defined members
+};
+```
+
+* *cl_interop*: Buffer constructed for interoperability with OpenCL cl\_mem objects
+
+```cpp
+struct cl_interop {
```

Comment:
The buffer/images are special cases, since according to the specification OpenCL interop buffers are technically tied to the original context. This modifies the normal behaviour of the buffer, which seems to justify a different property. In other SYCL objects with interop, the normal behaviour of the object is preserved (e.g. context or kernel interop)